example/webdriverio-test/test/specs/test.e2e.ts

```diff
@@ -2,78 +2,202 @@ import {driver} from '@wdio/globals';
 import {
   CONNECT_BUTTON,
   DISCONNECT_BUTTON,
+  JOIN_BUTTON,
   SHARE_SCREEN_BUTTON,
   SWITCH_CAMERA_BUTTON,
   TOGGLE_CAMERA_BUTTON,
+  TOGGLE_MICROPHONE_BUTTON,
   TOKEN_INPUT,
   URL_INPUT,
+  VIDEO_STATISTICS_RTC,
+  AUDIO_STATISTICS_RTC,
+  TOGGLE_STATISTICS_BUTTON,
+  NO_CAMERA_VIEW,
+  VIDEO_CELL,
 } from '../../../types/ComponentLabels';
 
 import {
-  platformJellyfishToken,
+  getBasePath,
+  getElement,
+  getWebsocketUrl,
   tapApp,
   tapButton,
   typeToInput,
+  compareInputValue,
 } from '../../utils';
 
+import {
+  AddPeerRequest,
+  Configuration,
+  ConfigurationParameters,
+  PeerDetailsResponseData,
+  Room,
+  RoomApiFp,
+} from '../../../server-api';
+
 import * as assert from 'assert';
 
+const createJellysifhRoom = async () => {
+  const configParam: ConfigurationParameters = {
+    accessToken: 'development',
+    basePath: `http://${getBasePath()}:${process.env.JELLYFISH_PORT}`,
+  };
+  const config = new Configuration(configParam);
+  const {createRoom} = RoomApiFp(config);
+  const createRoomFunction = await createRoom();
+  try {
+    const response = await createRoomFunction();
+    return response.data.data.room;
+  } catch (e) {
+    console.log(e);
+    return;
+  }
+};
+const addPeerToRoom = async (
+  roomId: string,
+  enableSimulcast: boolean = true,
+) => {
+  const configParam: ConfigurationParameters = {
+    accessToken: 'development',
+    basePath: `http://${getBasePath()}:${process.env.JELLYFISH_PORT}`,
+  };
+  const config = new Configuration(configParam);
+  const {addPeer} = RoomApiFp(config);
+  const addPeerRequest: AddPeerRequest = {
+    type: 'webrtc',
+    options: {enableSimulcast: enableSimulcast},
+  };
+  const addPeerFunction = await addPeer(roomId, addPeerRequest);
+  try {
+    const response = await addPeerFunction();
+    return response.data.data;
+  } catch (e) {
+    console.log(e);
+    return;
+  }
+};
+
+var peerDetail: PeerDetailsResponseData | undefined;
+var room: Room | undefined;
+
 describe('Walk through app', async () => {
+  it('create room and peer to obtain credentials', async () => {
```

Comment:
stworzenie roomu i peera powinno sie odbyć przed testami a nie być ich częścią
framework mocha posiada funkcje before i w niej to zrobić
dodatkowo jeżeli nie powiedzie ta częśc kodu to nie ma sensu puszczać reszte kodu, aktualnie reszta kodu i tak sie wykonuje